### PR TITLE
Sphe/feat/terraform

### PR DESCRIPTION
**Pull request by siphesihle1415**

### CODE REVIEW

File: .terraform/database.tf [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# DATABASE
+#########################################
+# RDS Database
+resource "random_password" "master_password" {
+  length           = 16
+  special          = true
+  override_special = "_!%^"
+}
+
+resource "aws_db_instance" "virtual_spider_zoo_db" {
```
Comment (seanvanwyk-bbd) [inline .terraform/database.tf]: Name is from your previous tf file

File: .terraform/database.tf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+resource "aws_secretsmanager_secret_version" "rds_credentials" {
+  secret_id     = aws_secretsmanager_secret.rds_credentials.id
+  secret_string = <<EOF
+  {
+    db_username = "${var.db_name}"
+    db_password = "${random_password.master_password.result}"
+    db_host     = "${aws_db_instance.virtual_spider_zoo_db.endpoint}"
+    db_port     = "${aws_db_instance.virtual_spider_zoo_db.port}"
```
Comment (seanvanwyk-bbd) [inline .terraform/database.tf]: Also used here

**Review by seanvanwyk-bbd — COMMENTED** [review]
Just a couple of terraform namings that should be updated, the terraform looks good

File: .terraform/database.tf (hunk 1 above)
Comment (siphesihle1415) [reply .terraform/database.tf]: Thanks for pointing that out.

**Review by seanvanwyk-bbd — APPROVED** [review]

**Review by kdcgovender — APPROVED** [review]

**Pull request merged**